web/ui/react-app/src/pages/graph/SeriesName.tsx

```diff
@@ -7,6 +8,23 @@ interface SeriesNameProps {
 }
 
 const SeriesName: FC<SeriesNameProps> = ({ labels, format }) => {
+  const toClipboard = (e: React.MouseEvent<HTMLSpanElement>) => {
+    let copyText = e.currentTarget.innerText || '';
+    if (copyText[0] === ',') {
+      copyText = copyText.slice(1);
+    }
+    navigator.clipboard
+      .writeText(copyText)
+      .then(() => {
+        toast.success('label selector   copied to clipboard', {
```

Comment:
Some extra spaces here, and "label matcher" would be the specific technical term for a single label:

```suggestion
        toast.success('Label matcher copied to clipboard.', {
```